pyanaconda/modules/localization/localization_interface.py

```diff
@@ -164,3 +156,31 @@ def KeyboardKickstarted(self) -> Bool:
         :return: True if keyboard command was seen in kickstart, otherwise False
         """
         return self.implementation.keyboard_seen
+
+    @emits_properties_changed
+    def SetKeyboardKickstarted(self, keyboard_seen: Bool):
```

Comment:
Just a note, I would like to eventually get rid of the `Kickstarted` properties.

Reply:
Yes, for now it became kind of consistent with locale settings of the localization module.